README.md

```diff
@@ -339,15 +339,13 @@ const search = instantsearch({
 })
 ```
 
-### ❌ Index
+### ✅ Index
 
 [Index references](https://www.algolia.com/doc/api-reference/widgets/index-widget/js/)
 
 `Index` is the component that lets you apply widgets to a dedicated index. It’s useful if you want to build an interface that targets multiple indices.
 
-Not compatible as Meilisearch does not support federated search on multiple indexes.
-
-If you'd like to see federated search implemented please vote for it in the [roadmap](https://roadmap.meilisearch.com/c/74-multi-index-search?utm_medium=social&utm_source=portal_share).
+Using this component, instant-meilisearch does an http-request for each different Index added. More http requests are made when using the [`RefinementList`](#✅-refinementlist) widget.
```

Comment:
Do you want to keep the `Index` with the uppercase?

Reply:
Yes since if the user uses the `Index` widget on the same index multiple times it still does that amount of HTTP requests. But you are right I'm going to improve that